### PR TITLE
Add swagger for Update.

### PR DESCRIPTION
**Pull request by MingshiPeng**
screen shot 2018-04-26 at 4 35 57 pm

### CODE REVIEW

File: app/server.py [hunk 1]
```diff
@@ -209,9 +209,33 @@ def delete_prod_info(prod_id):
...
+    parameters:
+        -   name: prod_id
+            in: path
+            description: ID of product information to be updated
+            type: integer
+            required: true
+        -   in: body
+            name: body
+            schema:
+                id: data
```
Comment (jockyxu1122) [inline app/server.py]: add a `ref` to Products definition under schema

**Review by nideyidayida — CHANGES_REQUESTED** [review]

File: app/server.py (hunk 1 above)
Comment (MingshiPeng) [reply app/server.py]: addressed

**Review by majia67 — APPROVED** [review]
Great!

**Review by nideyidayida — APPROVED** [review]

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180427-022549 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180427-022549") to [dev](https://console.bluemix.net/apps/1db994d8-c4f1-4076-85c0-7ff2674244e5?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:dev"), including commit bc3838dd8d240382f6a8fd69170a127b23f8b24c

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-183630 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-183630") to [prod](https://console.bluemix.net/apps/e25e653b-a50f-4831-b93e-d8f9f484a484?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:prod"), including commit bc3838dd8d240382f6a8fd69170a127b23f8b24c

**Pull request merged**